server/src/main/java/org/opensearch/extensions/ExtensionRequest.java

```diff
@@ -24,41 +25,53 @@
  */
 public class ExtensionRequest extends TransportRequest {
     private static final Logger logger = LogManager.getLogger(ExtensionRequest.class);
-    private ExtensionsManager.RequestType requestType;
+    private final ExtensionsManager.RequestType requestType;
+    private final Optional<String> uniqueId;
```

Comment:
It looks like there is no test class for `ExtensionRequest` so you should create one and include a test of this uniqueID, both in the `null` case (single arg constructor) or value present case (two arg constructor)